addons/knowledge/tests/test_knowledge_form_ui.py

```diff
@@ -65,3 +65,8 @@ def test_knowledge_main_flow(self):
         self.assertEqual(len(article_favorites), 2)
         self.assertEqual(article_favorites[0].article_id, private_article)
         self.assertEqual(article_favorites[1].article_id, workspace_article)
+
+    def test_knowledge_pick_emoji(self):
+        # remove existing articles to ease tour management
```

Comment:
As it is done also in test_knowledge_main_flow, could be done directly in setupclass (that was data cleaning is done once and outside of unit test itself)